dev/tools/svg2dart/test_assets/illegal_svg_multiple_roots.svg

```diff
@@ -0,0 +1,4 @@
+<svg id="svg_build_00" xmlns="http://www.w3.org/2000/svg" width="48px" height="48px" >
+</svg>
```

Comment:
you can just make this an empty SVG element (`<svg/>`)

Reply:
done